test/extended/setup.sh

```diff
@@ -147,6 +150,9 @@ function os::test::extended::setup () {
 
 	os::log::info "Creating image streams"
 	oc create -n openshift -f "${OS_ROOT}/examples/image-streams/image-streams-centos7.json" --config="${ADMIN_KUBECONFIG}"
+
+	os::log::info "Creating sample app"
+	oc create -n openshift -f "${OS_ROOT}/examples/sample-app/application-template-stibuild.json" --config="${ADMIN_KUBECONFIG}"
```

Comment:
discussed on IRC, it's needed by the broker (to have a valid template to instantiate) and creating it in the TC risks flakes due to tests running in parallel.